Updating official version to sbt one?

laughedelic opened the pull request:
In order to fix possible bugs and so on it'd be nice to merge the branch containing the new version including sbt to master as soon as possible. WDYT?
@eparejatobes @rtobes @marina-manrique

pablopareja committed:
starting with the adaptation to sbt

pablopareja committed:
updating build.sbt file

pablopareja committed:
updating build.sbt file

eparejatobes committed:
Update plugins.sbt

pablopareja committed:
first changes after configuring project with sbt

pablopareja committed:
making some progress on adapting code to new versions of dependencies

pablopareja committed:
making some progress on adapting code to new versions of dependencies

pablopareja committed:
finish refactoring of imports

pablopareja committed:
a few changes

pablopareja committed:
updating compiler.xml file

pablopareja committed:
fixing author javadoc tag syntax

pablopareja committed:
fixing package names adapting them to new policy

pablopareja committed:
fixing package references in XML executions template

pablopareja committed:
a few updates

pablopareja committed:
some updates

pablopareja committed:
making progress on the implementation of the program to import GenBank files

pablopareja committed:
adding a few fixes to Import GeneBank files program

pablopareja committed:
minor fix for product lines missing new line char

pablopareja committed:
some updates and fixes

pablopareja committed:
updating program that imports GeneBank files so that it adds an automatic ID to both genes and rnas. Some debug lines of code were also added to Export to EMBL files program

pablopareja committed:
minor updates and fixes in SolveOverlappings

pablopareja committed:
fixing package names of a few classes

pablopareja committed:
a few fixes regarding rna parsing

pablopareja committed:
adding check for empty rna annotation uniprot ID in ExportEmblFiles program

eparejatobes commented:
but this is just about the build process right?

pablopareja commented:
yeah, there might also be some minor fixes included for the exporting programs to GFF or EMBL formats

eparejatobes commented:
OK fine

pablopareja committed:
some updates

pablopareja committed:
some updates in build.sbt file

pablopareja committed:
some updates in build.sbt file

pablopareja committed:
adding fat jar file

pablopareja committed:
adding fat jar file

pablopareja committed:
adding debug line for stackTrace in FillUniprotData program

pablopareja committed:
adding line for .idea folder in gitignore file

pablopareja committed:
removing folder pushed by mistake

pablopareja committed:
adding an exponential timeout for the new neverending request for Uniprot data

pablopareja committed:
doing some clean up of old/obsolete stuff

pablopareja committed:
fix at GenerateCSVFiles

laughedelic commented:
This is relevant again :up:

laughedelic committed:
Cleaned up the project; updated deps; fixed fat-jar assembly

laughedelic commented:
Hey guys let's merge the `new/version` branch. I just cleaned up the project and updated deps in that branch, but I couldn't push it... Otherwise it seems to be fine (at least it compiles and produces fat-jar).
